src/styles/main.scss

```diff
@@ -1,3 +1,52 @@
-body {
-  margin: 0;
+@import './variables.scss';
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: start;
+  align-items: center;
+  gap: $gap-size;
+  width: $col-count * ($cell-size + $gap-size);
+  padding: $padding-size;
+  margin: auto;
+
+  &__day {
```

Comment:
Add 'cursor: pointer;' to the '.calendar__day' class to make the cursor a pointer on hover by default.